src/main/java/com/ibm/watson/developer_cloud/personality_insights/v2/model/ContentItem.java

```diff
@@ -25,77 +25,20 @@
 public class ContentItem extends GenericModel {
   private String charset;
   private String content;
-  private String contenttype;
+  private String contentType;
   
   @JsonAdapter(LongToDateTypeAdapter.class)
   private Date created;
-  private Boolean forward;
+  private boolean forward;
   private String id;
   private String language;
-  private String parentid;
-  private Boolean reply;
-  private String sourceid;
+  private String parentId;
+  private boolean reply;
+  private String sourceId;
   
   @JsonAdapter(LongToDateTypeAdapter.class)
   private Date updated;
-  private String userid;
-
-
-  /**
-   * Sets the character set of the text, for example, "UTF-8".
-   * 
-   * @param charset the new character set of the text, for example, "UTF-8"
-   * @return the content item
-   */
-  public ContentItem charset(String charset) {
```

Comment:
Although removing the Builder-style setters (`charset(String), content(String), ...`) and the lowercase getters and setters (`setContenttype(String)`, `getParentid(), ...`) is what we want to achive in the long term, these methods cannot be removed right now. This would break the code of our users who use the existing methods. According to [Semantic Versioning](http://semver.org/), breaking API changes should only be introduced in major releases (e.g. the 4.0 release of this API). In the meanwhile, please add `@Deprecated` annotations to the old methods instead of removing them completely.